api/tests/integration/domain/usecases/create-pro-organization-with-tags_test.js

```diff
@@ -199,145 +316,164 @@ describe('Integration | UseCases | create-pro-organization', function () {
+        .where({ organizationId: organizationInDB.id });
+      expect(organizationTagInDB.length).to.equal(2);
+
+      organizationTagInDB.forEach((value) => {
+        expect(value.organizationId).to.be.equal(organizationInDB.id);
+      });
+    }
   });
 
-  it('should rollback create pro organizations with tags and throw an error when there is more than one occurrence of the same organization', async function () {
+  it('should create organization invitation with role when role is specified', async function () {
     // given
-    const tooManyOccurencesOfTheSameorganizationWithTags = [
+    const organizationsWithInvitationRole = [
       {
         externalId: 'b200',
         name: 'Youness et Fils',
         provinceCode: '123',
         credit: 0,
         email: 'youness@example.net',
+        organizationInvitationRole: Membership.roles.ADMIN,
         locale: 'fr-fr',
-        tags: 'TagNotFound',
-      },
-      {
-        externalId: 'b202',
-        name: 'Mathieu Bâtiment',
-        provinceCode: '567',
-        credit: 20,
-        email: 'mathieu@example.net',
-        locale: 'fr-fr',
-        tags: 'TagNotFound',
+        tags: 'Tag1_Tag2',
+        createdBy: userId,
+        documentationUrl: null,
       },
       {
-        externalId: 'b202',
-        name: 'Mathieu Bâtiment',
-        provinceCode: '567',
-        credit: 20,
-        email: 'mathieu@example.net',
+        externalId: 'b300',
+        name: 'Andreia & Co',
+        provinceCode: '345',
+        credit: 10,
+        email: 'andreia@example.net',
+        organizationInvitationRole: Membership.roles.MEMBER,
         locale: 'fr-fr',
-        tags: 'TagNotFound',
+        tags: 'Tag2_Tag3',
+        createdBy: userId,
+        documentationUrl: null,
       },
     ];
 
     // when
-    const error = await catchErr(createProOrganizationsWithTags)({
+    await createProOrganizationsWithTags({
       domainTransaction,
-      organizations: tooManyOccurencesOfTheSameorganizationWithTags,
+      organizations: organizationsWithInvitationRole,
       organizationRepository,
       tagRepository,
       organizationTagRepository,
       organizationInvitationRepository,
     });
 
     // then
-    expect(error).to.be.instanceOf(ManyOrganizationsFoundError);
-    expect(error.message).to.be.equal('Une organisation apparaît plusieurs fois.');
-    const organizationsInDB = await knex('organizations').select();
-    expect(organizationsInDB.length).to.equal(0);
-    const organizationTagsInDB = await knex('organization-tags').select();
-    expect(organizationTagsInDB.length).to.equal(0);
+    for (const organizationInvitation of organizationsWithInvitationRole) {
+      const organizationInvitationInDB = await knex('organization-invitations')
+        .first('id', 'email', 'role')
+        .where({ email: organizationInvitation.email });
+      expect(organizationInvitationInDB.role).to.be.equal(organizationInvitation.organizationInvitationRole);
+    }
```

Comment:
**chipotage**
Je trouve toujours fatiguant à lire les boucles dans les tests, je suis obligée de chercher à quoi correspond chaque variables pour comprendre ce que ça fait. En plus ici il n'y a que 2 itérations à faire, ça prend moins de ligne et c'est plus lisible d'écrire sans boucler je trouve : 
```suggestion
const firstSavedOrganization = await knex('organization-invitations').where({ externalId: "b200"}).first();
expect(firstSavedOrganization.role).to.be.equal(Membership.roles.ADMIN);
const secondSavedOrganization = await knex('organization-invitations').where({ externalId: "b300"}).first();
expect(secondSavedOrganization.role).to.be.equal(Membership.roles.MEMBER);
```

Reply:
Ok, je peux modifier, c'est pas couteux.